dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/model/AnalyticsTable.java

```diff
@@ -62,6 +63,8 @@ public class AnalyticsTable extends Table {
   /** Tracked entity type of enrollments in analytics table. */
   private TrackedEntityType trackedEntityType;
 
+  @Getter @Setter private boolean tableDistributed = false;
+
```

Comment:
We are trying to keep `AnalyticsTable` and all other classes in db package immutable, so great to avoid @Setter / set methods.

Reply:
Right, I will see how to achieve it.